crowbar_framework/config/repos-cloud.yml

```diff
@@ -21,115 +21,6 @@ suse-12.1:
       url:
       smt_path: "SUSE/Updates/SLE-SERVER/12-SP1/x86_64/update"
       ask_on_error: false
-    cloud:
-      name: "Cloud"
-      required: "mandatory"
-      features: ["openstack"]
-      repomd:
-        tag: ["obsproduct://build.suse.de/SUSE:SLE-12-SP2:Update:Products:Cloud7/suse-openstack-cloud/7/cd/x86_64",
-              "obsproduct://build.suse.de/Devel:Cloud:7:Mitaka/suse-openstack-cloud/7/cd/x86_64",
-              "obsproduct://build.suse.de/Devel:Cloud:7:Staging/suse-openstack-cloud/7/cd/x86_64",
-              "obsproduct://build.suse.de/Devel:Cloud:7/suse-openstack-cloud/7/cd/x86_64"]
-        key_md5: ["9a62177e1c6852d48453ed3909956d6b", "6e2920076653964b7de9d6d0421955bb"]
-      url:
-      ask_on_error: false
-    suse-openstack-cloud-7-pool:
-      name: "SUSE-OpenStack-Cloud-7-Pool"
-      required: "mandatory"
-      features: ["openstack"]
-#      repomd:
-#        tag: "obsproduct://build.suse.de/SUSE:SLE-12-SP2:Update:Products:Cloud7/suse-openstack-cloud/7/POOL/x86_64"
-#        key_md5: "9a62177e1c6852d48453ed3909956d6b"
-      url:
-      smt_path: "SUSE/Products/OpenStack-Cloud/7/x86_64/product"
-      ask_on_error: false
-    suse-openstack-cloud-7-updates:
-      name: "SUSE-OpenStack-Cloud-7-Updates"
-      required: "mandatory"
-      features: ["openstack"]
-#      repomd:
-#        tag: "obsrepository://build.suse.de/SUSE:Updates:OpenStack-Cloud:7:x86_64/update"
-#        key_md5: "9a62177e1c6852d48453ed3909956d6b"
-      url:
-      smt_path: "SUSE/Updates/OpenStack-Cloud/7/x86_64/update"
-      ask_on_error: false
-    sle12-sp1-ha-pool:
-      name: "SLE12-SP1-HA-Pool"
-      required: "optional"
-      features: ["ha"]
-      repomd:
-        tag: "obsproduct://build.suse.de/SUSE:SLE-12-SP1:GA/sle-ha/12.1/POOL/x86_64"
-        key_md5: "9a62177e1c6852d48453ed3909956d6b"
-      url:
-      smt_path: "SUSE/Products/SLE-HA/12-SP1/x86_64/product"
-      ask_on_error: false
-    sle12-sp1-ha-updates:
-      name: "SLE12-SP1-HA-Updates"
-      required: "optional"
-      features: ["ha"]
-      repomd:
-        tag: "obsrepository://build.suse.de/SUSE:Updates:SLE-HA:12-SP1:x86_64/update"
-        key_md5: "9a62177e1c6852d48453ed3909956d6b"
-      url:
-      smt_path: "SUSE/Updates/SLE-HA/12-SP1/x86_64/update"
-      ask_on_error: false
-    suse-enterprise-storage-2_1-pool:
```

Comment:
And relevant PR is https://github.com/crowbar/crowbar-core/pull/483 (we'll need to rebase one of these obviously)